app/models/nonbillable_account.rb

```diff
@@ -27,6 +28,12 @@ def to_s(with_owner = false, flag_suspended = true, with_facility: false)
 
   private
 
+  # This insures that NonbillableAccount#price_groups isn't empty, in the case
+  # the user doesn't have a price group
+  def set_price_group
+    price_group_members.create(price_group: PriceGroup.base, type: "AccountPriceGroupMember")
```

Comment:
@jossim what would you think about creating a class method `PriceGroup#nonbillable` that returns `PriceGroup.base`.  I think this would make the intent more obvious and reduce duplication.